tests/inventory-product.sql

```diff
@@ -0,0 +1,35 @@
+SET 'auto.offset.reset' = 'earliest';
+
+CREATE STREAM inventory_stream (
```

Comment:
Do we need to include this test as part of connector's repo?

Reply:
to me, this seems a bit oddly placed in the connector's repo as connector has nothing to do with the ksql queries. IMO this should be included probably in repo like https://github.com/confluentinc/kafka-tutorials , wdyt?

Reply:
Reasoning for including it here was to show interested users how to process the data in ksqlDB, and it validates that it can be done.  Otherwise, how do we ensure end-to-end that the data written can be processed by ksqlDB?  It doesn't necessitate a full Kafka tutorial, it's more of a functional test for the datagen set.  I think it does belong here, but if it helps with demarkation, what if we rename the folder from `tests` to something with ksqlDB in the name like `tests/consume-from-ksqlDB`?